lib/init/index.js

```diff
@@ -7,71 +7,71 @@ const pEachSeries = require("p-each-series");
 
 const runPrettier = require("../utils/run-prettier");
 
-const entryTransform = require("./transformations/entry/entry");
-const outputTransform = require("./transformations/output/output");
+const amdTransform = require("./transformations/other/amd");
```

Comment:
Maybe we can consider these eslint rules:
- https://eslint.org/docs/rules/sort-vars
- https://eslint.org/docs/rules/sort-keys

Although it could be a killer eheh

Reply:
Sure! I'll do it in a separate PR if we all agree on adding the rule.